src/HttpHandler/Guzzle7HttpHandler.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
This file needs a copyright notice.

Reply:
Note to self: That's what happens if you just copy and paste 😅.

I added one to the Guzzle6Handler as well (with 2015, since it was introduced in that year)

Reply:
Thanks! And thank you for catching the missing copyright in the other file! I never noticed. 😆